packages/fast-usdc/src/exos/status-manager.js

```diff
@@ -325,19 +324,10 @@ export const prepareStatusManager = (
       /**
        * Mark a transaction as `FORWARDED`
        *
-       * @param {EvmHash | undefined} txHash - undefined in case mint before observed
-       * @param {NobleAddress} nfa
-       * @param {bigint} amount
+       * @param {EvmHash} txHash - undefined in case mint before observed
```

Comment:
the "undefined in case..." docstring looks outdated

Reply:
I suppose the preconditions for `forwarded(...)` are in a state transition diagram somewhere, but it would be nice to have them here locally.

In particular, this would throw if you call it twice on the same txHash, right?